es-design-system/pages/molecules/es-form-msg.vue

```diff
@@ -8,17 +8,35 @@
                 bootstrap-vue alert
             </b-link>
         </p>
-        <es-button @click="fakeFormMsg()">
-            Show Success
+
+        <es-button
+            @click="showSuccess = true">Show Success
```

Comment:
Nit: Can we change this to "Show success" to match our [writing guidelines about EsButton captions](https://design.energysage.dev/molecules/es-button) being sentence case?